Tutorial/objects 1144 mac setup

sdomula opened the pull request:
I added a tutorial to install the DevKit on macOS and GNU/Linux. Now I hope to get input from you
on where I should extend/fix the install tutorial.

I suggest to have a look at the rendered documentation which I linked below.

I made changes to the following section:
* [main README DevKit section](https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/tree/tutorial/OBJECTS-1144_mac-setup#smart-cosmos-devkit)

and added these documents:
* [gettings started tutorial landing page](https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/tutorial/OBJECTS-1144_mac-setup/tutorials/getting-started.adoc#getting-started)
* [installation prerequisites](https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/tutorial/OBJECTS-1144_mac-setup/tutorials/prerequisites.adoc#prerequisites)
* [devkit setup](https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/tutorial/OBJECTS-1144_mac-setup/tutorials/install-devkit.adoc#install-the-devkit)

Commit:
inital version of the installation tutorial

Commit:
try to fix cross ref

Commit:
try other ref

Commit:
add devkit start instructions

Commit:
add instuctions for GNU/Linux

Commit:
clean up

Commit:
refer back to next steps

Commit:
rearrange the next steps

sdomula committed:
add line

Commit:
reorganize the install tutorials

Commit:
typo

Commit:
links

Commit:
refer to prerequisites in devkit set up guide

Commit:
use memory not RAM to match the docker pref window

Commit:
add to

Commit:
only use 127.0.0.1 and not 0.0.0.0

jbauerrfid reviewed (approved):
LGTM

Commit:
ui will be removed so remving the port

sdomula committed:
Merge branch 'master' into tutorial/OBJECTS-1144_mac-setup

Commit:
add docker starting instructions

andreassiegel reviewed (approved):
Only some minor remarks, generally looks good.

Commit:
use brew cask to install docker

If using `brew install docker` it only installes
the docker cli and not the engine.

Commit:
tap the caskroom

Commit:
add instuctions to enable the caskroom

Commit:
typo

Commit:
fix headings

Commit:
use $ before shell commands

Commit:
brew->Homebrew

Commit:
use <kbd>...</kbd>

Commit:
use kbd makro instead of md syntax

Commit:
improved docker-engine start insructions.

Commit:
fix typo

Commit:
clean up

Commit:
reduce min Memory to 5GB

Commit:
do bind to 0.0.0.0

Commit:
revert application.yml port changes

Commit:
fix auth-server port-forwarding

Commit:
link to List of Ports table

Commit:
minor changes

Commit:
improve docker icon descriptions

Commit:
add missing space

Commit:
Fix link for list of ports
